src/components/20-molecules/dropdown/index.scss

```diff
@@ -91,6 +91,8 @@ axa-dropdown {
     padding: 0 40px 0 20px;
     height: $dropdown-height;
     box-sizing: border-box;
+    text-overflow: ellipsis;
```

Comment:
This is fine and reduces the impact of the 'problem' the ticket reported. However, not sure whether it's right to close the original ticket with a - in the view of the requester - only partial solution.